docs/docs/examples/cluster/CLOUD_PROVIDERS.md

```diff
@@ -1,9 +1,7 @@
 # Provisioning Kubernetes clusters on cloud-based providers
+
 Here are installation and configuration instructions for cloud providers:
+
 * [Amazon EKS](EKS_SETUP.md) 
 * [Google GKE](GKE_SETUP.md)
 * [Azure AKS](AKS_SETUP.md)
-
-***
```

Comment:
These links are not necessary as we have navigation provided by mkdocs now. They are removed from all the pages.